epregressions/runtests.py

```diff
@@ -223,10 +223,10 @@ def add_or_modify_output_sqlite(idf_text, force_output_sql: ForceOutputSQL,
                 break
         if has_sqlite_object:
             import re
-            RE_SQLITE = re.compile('Output:SQlite\s*,(?P<Option>[^,;]*?)\s*(?P<TabularUnitConv>,[^,;]*?\s*)?;',
+            RE_SQLITE = re.compile(r'Output:SQlite\s*,(?P<Option>[^,;]*?)\s*(?P<TabularUnitConv>,[^,;]*?\s*)?;',
                                    re.IGNORECASE)
             if force_output_sql_unitconv == ForceOutputSQLUnitConversion.NOFORCE:
-                idf_text = RE_SQLITE.sub('Output:SQLite,\n    {}\g<TabularUnitConv>;\n'.format(force_output_sql.value),
+                idf_text = RE_SQLITE.sub(r'Output:SQLite,\n    {}\g<TabularUnitConv>;\n'.format(force_output_sql.value),
```

Comment:
This is why I omitted the `r`. Somehow I thought you couldn't `.format` a string with a `r`.

Seems like it works though.

```
In [1]: r'Output:SQLite,\n {}\g<TabularUnitConv>;\n'.format('REPLACEMENT')
Out[1]: 'Output:SQLite,\\n REPLACEMENT\\g<TabularUnitConv>;\\n'
```